development/tools/cmd/synchronizer/syncomponent/component.go

```diff
@@ -0,0 +1,104 @@
+package syncomponent
+
+import (
+	"log"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const defaultOutOfDateDays = 3
```

Comment:
this is surprise, because in main it looks like default is 0

Reply:
fixed, now default value is setup in main